examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,25 @@
+"""
+Contours
+--------
```

Comment:
```suggestion
Contour (tabular data)
----------------------
```
I am recommending this change because it is likely that we'll add another gallery example for contour plots of gridded data.